chart/templates/flower/flower-deployment.yaml

```diff
@@ -81,19 +81,36 @@ spec:
               containerPort: {{ .Values.ports.flowerUI }}
           livenessProbe:
             failureThreshold: 10
-            httpGet:
-              path: /
-              port: {{ .Values.ports.flowerUI }}
+            exec:
+              command:
+                - curl
+                {{- if or .Values.flower.secretName .Values.flower.basicAuth }}
+                - "-H"
+                - "Authorization: Basic $AIRFLOW__CELERY__FLOWER_BASIC_AUTH"
+                {{- end }}
+                - "localhost:5555"
```

Comment:
Does this option have no effect on this URL?
https://github.com/apache/airflow/blob/5a396835e6fab50a8222c865056a66a6986f0265/chart/values.yaml#L547